packages/vow/README.md

```diff
@@ -77,6 +77,67 @@ const { watch, makeVowKit } = prepareVowTools(vowZone);
 // Vows and resolvers you create can be saved in durable stores.
 ```
 
+## VowTools
+
+VowTools are a set of utility functions for working with Vows in Agoric smart contracts and vats. These tools help manage asynchronous operations in a way that's resilient to vat upgrades, ensuring your smart contract can handle long-running processes reliably.
+
+### Usage
+
+VowTools are typically prepared in the start function of a smart contract or vat and passed in as a power to exos.
+
+
+```javascript
+import { prepareVowTools } from '@agoric/vow/vat.js';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+
+export const start = async (zcf, privateArgs, baggage) => {
+  const zone = makeDurableZone(baggage);
+  const vowTools = prepareVowTools(zone.subZone('vows'));  
+
+  // Use vowTools here...
+}
+```
+
+### Available Tools
+
+#### `when(vowOrPromise)`
+Waits for a Vow or Promise to resolve and returns its value.  It can't handle disconnections due to vat upgrades.
```

Comment:
```suggestion
Returns a Promise for the fulfillment of the very end of the `vowOrPromise` chain.  It can retry disconnections due to upgrades of other vats, but cannot survive the upgrade of the calling vat.
```